src/rest_framework_jwt/blacklist/permissions.py

```diff
@@ -0,0 +1,20 @@
+
+from rest_framework.permissions import BasePermission
+
+from rest_framework_jwt.blacklist.models import BlacklistedToken
+from rest_framework_jwt.settings import api_settings
+
+
+class IsAuthenticatedAndNotBlacklisted(BasePermission):
+    message = 'You are not authenticated or have been blacklisted.'
```

Comment:
Isn't this permission doing too much? What happens if users of this library want to combine the `blacklist` permission with their customized `IsAuthenticated` permissions?